consumer.go

```diff
@@ -277,7 +282,7 @@ func (child *partitionConsumer) dispatcher() {
 				select {
 				case <-child.dying:
 					close(child.trigger)
-				case <-time.After(10 * time.Second):
+				case <-time.After(child.conf.Consumer.Retry.Backoff):
```

Comment:
In hindsight I think we can actually simplify this loop quite a bit - simply arriving here (via a message on `child.trigger`) indicates that some worker received an error or NotLeader or whatever, so we should back off right away, as dispatching right away probably won't work anyways.

Given that, I think the `<-time.After` case should replace the main `default` case, and this inner select can go away entirely. Makes sense?